.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/marketplace/actions/close-stale-issues
+name: "Stale issues"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # issue configuration
+          stale-issue-message: "This issue is stale because it has been open 60 days with no activity. Remove the stale label or add a comment or this will be closed in 14 days."
```

Comment:
```suggestion
          stale-issue-message: "This issue is stale because it has been open 60 days with no response activity. Remove the stale label, add a comment, or this will be closed in 14 days."
```